.github/workflows/swift-validate.yml

```diff
@@ -23,12 +23,12 @@ jobs:
         run: swiftlint lint --reporter github-actions-logging
 
   test:
-    name: Test on ${{ matrix.os }} with ${{ matrix.swift }} 
+    name: Test on ${{ matrix.os }} with ${{ matrix.swift }}
     timeout-minutes: 40
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        swift: ["5.9", "5.10"]
+        swift: ["5.10", "6.0"]
```

Comment:
Wait before https://github.com/swift-actions/setup-swift has support for swift 6